These were probably always wrong, but it only fails in public
usage (e.g. from gjs) from now on.

diff --git a/mfbt/UniquePtrExtensions.h b/mfbt/UniquePtrExtensions.h
index 565d4bf1f..d5a7b1502 100644
--- a/mfbt/UniquePtrExtensions.h
+++ b/mfbt/UniquePtrExtensions.h
@@ -17,6 +17,10 @@
 #include "mozilla/fallible.h"
 #include "mozilla/UniquePtr.h"
 
+#ifndef XP_UNIX
+#define XP_UNIX
+#endif
+
 #ifdef XP_WIN
 #  include <cstdint>
 #endif
diff --git a/mozglue/baseprofiler/public/BaseProfilingCategory.h b/mozglue/baseprofiler/public/BaseProfilingCategory.h
index d6f7148c9..3091ecc5f 100644
--- a/mozglue/baseprofiler/public/BaseProfilingCategory.h
+++ b/mozglue/baseprofiler/public/BaseProfilingCategory.h
@@ -12,7 +12,7 @@
 
 #include <cstdint>
 
-#include "ProfilingCategoryList.h"
+#include "js/ProfilingCategoryList.h"
 
 namespace mozilla {
 namespace baseprofiler {
